src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -501,7 +500,7 @@ private function createFirewall(ContainerBuilder $container, string $id, array $
         $configuredEntryPoint = $defaultEntryPoint;
 
         // authenticator manager
-        $authenticators = array_map(fn ($id) => new Reference($id), $firewallAuthenticationProviders);
+        $authenticators = array_map(fn ($id) => new Reference($id), $firewallAuthenticationProviders, []);
```

Comment:
The added array’s purpose is to reset `$authenticators`’s key (cf https://www.php.net/manual/en/function.array-map.php#refsect1-function.array-map-returnvalues) so that `testConfigureCustomAuthenticator` doesn’t fail.